### PR TITLE
feat: chart prototypes

### PR DESCRIPTION
**Pull request by madhuravius**
requires metrictunnel and other changes pushed to prod:

DB

image

image
image

App:

image

image

### CODE REVIEW

**Comment by joeclayallday** [comment]
Make responsive charts

https://github.com/aptible/app-ui/assets/4295811/9bfae578-a670-4b7e-9644-c346794ed86d

**Comment by joeclayallday** [comment]
Update Chart Styling

Screen Shot 2023-07-11 at 8 49 02 AM

File: src/app/app.tsx [hunk 1]
```diff
@@ -8,8 +8,36 @@ import { ModalPortal, StandaloneErrorBoundary } from "@app/ui";
 import { ftuxRouter, router } from "./router";
 import { selectOrigin } from "@app/env";
 import { CookieNotice } from "@app/ui/shared/cookie-notice";
+import {
```
Comment (madhuravius) [inline src/app/app.tsx:11]: if merging, ensure this is isolated to the chart component and use lazy loading / treeshaking, or we probably can't merge until we have a charting solution

**Comment by joeclayallday** [comment]
https://github.com/aptible/app-ui/assets/4295811/1b262d4f-7354-44fe-acd2-a16a246ee702

**Pull request closed without merging**